packages/identitykit/src/libs/react/provider.tsx

```diff
@@ -55,14 +55,14 @@ export const IdentityKitProvider = <T extends IdentityKitAuthType>({
   const { signerClient, setSignerClient } = useCreateIdentityKit({
     selectedSigner: selectedSigner ?? savedSigner,
     signerClientOptions: {
-      ...signerClientOptions,
       idleOptions: {
-        ...signerClientOptions?.idleOptions,
+        disableIdle: false,
+        idleTimeout: 14_400_000,
         onIdle: () => {
-          signerClientOptions?.idleOptions?.onIdle?.()
           logoutByIdle()
         },
       },
+      ...signerClientOptions,
```

Comment:
but this way if user changes idleTimeout, his options will override ours and logout will not work

Reply:
ah, yes
I was confused a bit with all these manipulations, but I got it now.